### PR TITLE
Fix status and tracing commands

### PR DESCRIPTION
**Pull request by kraman**
Connect to strongloop-internal/scrum-nodeops#627

### CODE REVIEW

File: bin/sl-meshctl.txt [hunk 1]
```diff
@@ -84,6 +84,10 @@ services using `ls` or `status`.
         a grace period for any existing connections to finish. "Hard" stops
         kill the supervisor and its workers with `SIGTERM`.
 
+  tracing-start SVC       Restart all workers with tracing on.
```
Comment (sam-github) [inline bin/sl-meshctl.txt]: Eventually, this should be `SVC | SVC.EXEC`, but this is fine for strong-pm, where its the same.

**Comment by sam-github** [comment]
LGTM

File: bin/sl-meshctl.txt (hunk 1 above)
Comment (kraman) [inline bin/sl-meshctl.txt]: yah, that was my intention. altough it accepts SVC.EXEC I didnt want to complicate the help info

**Pull request merged**